### PR TITLE
Update README for 1.12 release

### PR DESCRIPTION
**Pull request by wonjoolee95**
Update README for 1.12 release

Confirmed that the following images/wheels are currently available:
- gcr.io/tpu-pytorch/xla:r1.12_3.7
- gcr.io/tpu-pytorch/xla:r1.12_3.8_tpuvm
- gcr.io/tpu-pytorch/xla:r1.12_3.7_cuda_11.2
- https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.12-cp37-cp37m-linux_x86_64.whl
- https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-1.12-cp38-cp38-linux_x86_64.whl
- https://storage.googleapis.com/tpu-pytorch/wheels/colab/torch_xla-1.12-cp37-cp37m-linux_x86_64.whl
- https://storage.googleapis.com/tpu-pytorch/wheels/cuda/112/torch_xla-1.12-cp37-cp37m-linux_x86_64.whl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,19 +98,19 @@ Follow these steps to train a PyTorch model with Docker on a Cloud TPU:
     ```
 
     ```Shell
-    (vm)$ docker pull gcr.io/tpu-pytorch/xla:r1.11_3.7
+    (vm)$ docker pull gcr.io/tpu-pytorch/xla:r1.12_3.7
```
[the remark this answers is not in the document]
Comment (wonjoolee95) [reply README.md:101]: Yep this is available now, doing `docker pull gcr.io/tpu-pytorch/xla:r1.12_3.7` will work.

**Comment by wonjoolee95** [comment]
Some of these should wait until the 2vm and 1vm are ready.

**Comment by wonjoolee95** [comment]
> Can we verify every wheel/docker listed here is availiable?

@JackCaoG, updated the PR summary above to confirm all these images/wheels are currently available.

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Pull request merged**